accepted/0000-npm-diff.md

```diff
@@ -18,13 +18,13 @@ Introduce a new `npm diff` command that accepts one or more package specs and
 uses these specs to compare files and print diff patches. Contents are fetched
 using `pacote`, accepting any type of specs that the npm installer understands.
 
-### npm diff `<spec>`
+### npm diff --diff=`<spec>`
```

Comment:
the double "diff" reads pretty awkwardly, but i don't have a better bikeshed color to suggest.

Reply:
How about arguments like this?

```
npm diff `<spec-a>` `<spec-b>`
```

Reply:
that’s just positional arguments (there’s no backticks convention in cli args), which I’d prefer, but I’m led to believe there’s a technical barrier to that.

Reply:
yes, we started there but we switched to named arguments in order to avoid the ambiguity between specs and file paths to filter